lib/platform/azure/index.ts

```diff
@@ -416,6 +416,19 @@ export async function createPr({
       pr.pullRequestId
     );
   }
+  if (platformOptions?.azureAutoApprove) {
+    pr = await azureApiGit.createPullRequestReviewer(
+      {
+        reviewerUrl: pr.createdBy.url,
+        vote: 10,
```

Comment:
tiny nit: this is a magic number. I know it comes from Azure DevOps, but can you either leave a comment where you found it ([here?](https://docs.microsoft.com/en-us/rest/api/azure/devops/git/pull%20request%20reviewers/create%20pull%20request%20reviewer?view=azure-devops-rest-6.0)) or make this a constant with a good name.

Reply:
Good point @JamieMagee. I used the source you linked for that magic number, and fixed it by adding AzurePrVote enum to make it clear what it means.